runtime/manta/src/lib.rs

```diff
@@ -151,12 +150,35 @@ parameter_types! {
 		})
 		.avg_block_initialization(AVERAGE_ON_INITIALIZE_RATIO)
 		.build_or_panic();
-	pub const SS58Prefix: u8 = manta_primitives::constants::MANTAPC_SS58PREFIX;
+	pub const SS58Prefix: u8 = manta_primitives::constants::MANTA_SS58PREFIX;
+}
+
+// Don't allow permission-less asset creation.
+pub struct MantaFilter;
+impl Contains<Call> for MantaFilter {
+	fn contains(c: &Call) -> bool {
+		match c {
+			Call::Timestamp(_)
+			| Call::ParachainSystem(_)
+			| Call::Authorship(_)
+			| Call::Sudo(_)
+			| Call::Multisig(_)
+			| Call::Scheduler(_)
```

Comment:
Actually why do we need the `Scheduler` when we don't have governace here yet.
Also is `Balances` needed in the shell chain ?

Reply:
Yeah, Balance is needed to help testing (at least do no harm). We could ban `Scheduler`.